android/app/src/main/java/org/fossasia/openevent/fragments/DayScheduleFragment.java

```diff
@@ -39,7 +39,7 @@
 /**
  * Created by Manan Wason on 17/06/16.
  */
-public class DayScheduleFragment extends Fragment {
+public class DayScheduleFragment extends Fragment implements SwipeRefreshLayout.OnRefreshListener {
```

Comment:
@gogeta95 You can remove this as we have only one listener and detecting calls there works fine for now. There is no overhead for this as well